pkg/adapter/rule.go

```diff
@@ -67,12 +82,12 @@ func (rule *Rule) Map(ref, opt string) (string, error) {
 		}
 	case CacheTagSuffix:
 		for _, item := range rule.items {
-			if item.CacheTagSuffix != "" {
-				if strings.HasSuffix(ref, item.CacheTagSuffix) {
+			if item.CacheTag != "" {
+				if strings.HasSuffix(ref, item.CacheTag) {
```

Comment:
We should check the original tag, instead of only checking the reference suffix.